CLAUDE.md

```
@@ -588,10 +588,44 @@ git commit -m "Brief description of change"
+4. **Add Python API support** (see below)
+5. Add tests in `tests/test_newcommand.py` and `tests/test_api.py`
+6. Check complexity: `xenon --max-absolute=A geoparquet_io/core/newcommand.py`
+
+### Adding Python API for New Commands
+
+**CRITICAL: Every new CLI command must have a corresponding Python API.**
+
+The Python API is equally important as the CLI. Users expect to be able to use gpio programmatically.
+
+1. **Add a method to `Table` class** in `api/table.py`:
+   ```python
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)